examples/llava/requirements.txt

```diff
@@ -2,4 +2,4 @@
 --extra-index-url https://download.pytorch.org/whl/cpu
 pillow~=10.2.0
 torch~=2.2.1
-torchvision==0.17.1
+torchvision!=0.17.1
```

Comment:
Is this intentional? I don't know what that syntax does in pip but it seems like it should be `~=` instead.

Reply:
Thanks for catching this, it was a mistake